vllm/model_executor/layers/activation.py

```diff
@@ -98,6 +101,9 @@ def extra_repr(self) -> str:
 
 class NewGELU(CustomOp):
 
+    def __init__(self):
+        super().__init__("gelu_new")
```

Comment:
can we make it a class attribute? and there should be some registry in the `CustomOp` class, so that we can call some functions to list all custom ops.